ius-release.spec

```diff
@@ -17,6 +17,9 @@ Source21:       ius-7.repo
 Source22:       ius-testing-7.repo
 Source23:       ius-archive-7.repo
 
+Source30:	ius-6-x86_64.cfg
+Source31:	ius-7-x86_64.cfg
```

Comment:
These lines have tabs, swap them for spaces to be consistent with the rest of the spec file.